.solcover.js

```diff
@@ -1,4 +1,4 @@
-const skipFiles = ['bancor', 'ens', 'erc1056']
+const skipFiles = ['bancor', 'ens', 'erc1056', 'arbitrum', 'tests/arbitrum']
```

Comment:
We could still include these, and just add a bunch more tests...

Reply:
(I would suggest leaving this as it is for now though, as it makes little sense to test thirdparty/mock code that we don't use, and coverage would naturally test the parts that we do use)